### PR TITLE
Add new formula pass-csv

### PR DESCRIPTION
**Pull request by lahr**

### CODE REVIEW

File: Formula/pass-csv.rb
Comment (SimplyDanny) [inline Formula/pass-csv.rb]: The class has to inherit from `PassExtensionFormula` instead of the common `Formula`. Otherwise the test will not run.

File: Formula/pass-csv.rb
Comment (SimplyDanny) [inline Formula/pass-csv.rb]: For the name of the password `"foo/bar"` there is the variable `@@example_password`. Also, you may quote the result string using `%(...)` to avoid all the escaping:
```suggestion
      assert_equal %("name","key1"\n"foo/bar","-"\n), File.read("output.txt")
```

**Review by SimplyDanny — CHANGES_REQUESTED** [review]
Thank you for your contribution and the nice pass extension!

Let me request just two changes before I hit the merge button. You can run the defined test via `brew test pass-csv` after you have installed the formula locally.

File: Formula/pass-csv.rb
Comment (lahr) [reply Formula/pass-csv.rb]: Ah yes, I forgot to change it back after I tested something.

**Review by SimplyDanny — APPROVED** [review]

**Pull request merged**
